ReleaseProcessAutomation/Jira/ServiceFacadeImplementations/JiraIssue.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// See the NOTICE file distributed with this work for additional information
+// regarding copyright ownership.  rubicon licenses this file to you under 
+// the Apache License, Version 2.0 (the "License"); you may not use this 
+// file except in compliance with the License.  You may obtain a copy of the 
+// License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software 
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT 
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the 
+// License for the specific language governing permissions and limitations
+// under the License.
+//
+
+using System;
+using System.Collections.Generic;
+
+namespace ReleaseProcessAutomation.Jira.ServiceFacadeImplementations
+{
+  public class JiraIssue
```

Comment:
I don't know how much benefit nullability info for this DTOs provide. Because whenever we use the (see JiraIssueService@42) we need to check for null all the time. Maybe it is better to remove it here and just go with the REST Api should return correct results. Or check them once when receving them.

Reply:
Quite understandable as this was simply a change to enable me to work on different aspects. I fully agree on removing nullability as it brings no clear advantages IMO. 

I do, however, not know the best way of removing nullability in this case. Leave them as properties and initialize them? Create constructors? What would be the proper way?

Reply:
Probably the best way is to add a `#nullable disable` directive at the top of the files that contain DTOs. We could also do `= default!;` initialization but that is more verbose.